docs/changelog/88064.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 88064
+summary: Fix typo in TransportForceMergeAction and TransportClearIndicesCacheAction
+area: Doc
+type: enhancement
+issues: []
```

Comment:
Hello @mushao999, thanks for this javadocs corrections!
I believe javadocs are not considered to be a docs are. I have marked this change as `>non-issue` do you mind removing this yaml file to make the build pass?

Reply:
> Hello @mushao999, thanks for this javadocs corrections! I believe javadocs are not considered to be a docs are. I have marked this change as `>non-issue` do you mind removing this yaml file to make the build pass?

@idegtiarenko Thanks for you reply, yaml file has been removed.